### PR TITLE
Savva Dzhumaliev Mock 1

### PR DESCRIPTION
**Pull request by EasileleM**

### CODE REVIEW

**Comment by EasileleM** [comment]
https://savva-dzhumaliev-task1.firebaseapp.com/

File: doc1/public/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style/reset/reset.css">
+    <link rel="stylesheet" href="style/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (vadeneev) [inline doc1/public/index.html]: There should be better place for viewport meta tag

File: doc1/public/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
```
Comment (vadeneev) [inline doc1/public/index.html]: check this one https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

File: doc1/public/index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    <link rel="stylesheet" href="style/reset/reset.css">
+    <link rel="stylesheet" href="style/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Design Lab</title>
+  </head>
+  <body>
+    
+    <div class="wrapper">
+        <header class="container header">
```
Comment (vadeneev) [inline doc1/public/index.html]: From my point of view - 'container' is so indistinct and too general name for block, doesn't it look so for you?

File: doc1/public/index.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+            <img class="header__logo" src="assets/header/Ellipse_1_copy_11.png" alt="Ellips">
+            <div class="header__text-block">
+                <h1 class="container_header-text header__text-block_header-text">Working <br> with design lab</h1>
```
Comment (vadeneev) [inline doc1/public/index.html]: It is not good practice to use element of parent's block inside its children:
https://ru.bem.info/methodology/key-concepts/#%D0%B1%D0%BB%D0%BE%D0%BA

File: doc1/public/index.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                  We help you to fill the gap between development, <br>
+                  design and product management.
+                </p>
+            </div>
+            <a href="#aboutUs" class="header__wrapper-mouse-icon">
+              <img class="header__mouse-icon" src="assets/header/Shape_1.svg" alt="mouse icon">
+            </a>
+        </header>
+        <div class="container" id="aboutUs">
+          <div class="container__border-text-block container__border-text-block_black about-us">
```
Comment (vadeneev) [inline doc1/public/index.html]: how about container__text-block_bordered ?
and what is text-block here, maybe it is an article ?

File: doc1/public/index.html [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    <title>Design Lab</title>
+  </head>
+  <body>
+    
+    <div class="wrapper">
+        <header class="container header">
+            <img class="header__logo" src="assets/header/Ellipse_1_copy_11.png" alt="Ellips">
+            <div class="header__text-block">
+                <h1 class="container_header-text header__text-block_header-text">Working <br> with design lab</h1>
+                <p class="container_text header__text-block_text">
```
Comment (vadeneev) [inline doc1/public/index.html]: text-block_text ?

Is text modifier here ?

File: doc1/public/index.html [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                  design and product management.
+                </p>
+            </div>
+            <a href="#aboutUs" class="header__wrapper-mouse-icon">
+              <img class="header__mouse-icon" src="assets/header/Shape_1.svg" alt="mouse icon">
+            </a>
+        </header>
+        <div class="container" id="aboutUs">
+          <div class="container__border-text-block container__border-text-block_black about-us">
+            <h2 class="container_header-text container_header-text-black">About us</h2>
```
Comment (vadeneev) [inline doc1/public/index.html]: black inside black - probably smth went wrong  for this moment

File: doc1/public/index.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+                We start each project by understanding your business goals, <br> 
+                the user’s needs and the product requirements. <br>
+                Then we help translate them into effective and elegant solutions. <br>
+            </p>
+          </div>
+        </div>
+
+        <div class="container services-wrap">
+            <div class="container__border-text-block container__border-text-block_white services">
+                <h2 class="container_header-text container_header-text-white">Services</h2>
```
Comment (vadeneev) [inline doc1/public/index.html]: white inside white

File: doc1/public/index.html [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+                </p>
+            </div>
+        </div>
+
+        <div class="container">
+            <div class="container__border-text-block container__border-text-block_black what-we-do">
+                <h2 class="container_header-text container_header-text-black what-we-do__main-header">What we do</h2>
+                <img src="assets/what-we-do/Shape_2.svg" alt="design" class="what-we-do__icon">
+                <h2 class="container_header-text what-we-do__sub-header">Design</h2>
+                <p class="container_text what-we-do__text">
```
Comment (vadeneev) [inline doc1/public/index.html]: So we are mixing two entities on one html's top level entity, there should be better way to achieve desired result

File: doc1/public/index.html [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+              <a href="#" class="our-works__ceil_text">Market</a>
+            </div>
+          </div>
+          <div class="our-works our-works_bottom">
+              <div class="our-works__ceil our-works__ceil_left">
+                  <a href="#" class="our-works_topworkz-image our-works_worktop"></a>
+                  <a href="#" class="our-works__ceil_text">Worktop</a>
+                </div>
+                <div class="our-works__ceil our-works__ceil_mid">
+                  <a href="#" class="our-works_topworkz-image our-works_lable"></a>
```
Comment (vadeneev) [inline doc1/public/index.html]: Maybe label ?

File: doc1/public/style/style.css [hunk 11]
```diff
@@ -0,0 +1,356 @@
...
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: column;
+  background-size: cover;
+}
+
+.container_header-text {
+  font-size: 3.15vh;
```
Comment (vadeneev) [inline doc1/public/style/style.css]: It is great way to achieve responsive behavior, however: 
image

it what was not expected, or am i wrong ?

**Review by vadeneev — CHANGES_REQUESTED** [review]

File: doc1/public/style/style.css (hunk 11 above)
Comment (EasileleM) [reply doc1/public/style/style.css]: I'm wrong here, ty

File: doc1/public/index.html [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+  <link rel="stylesheet" href="style/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <img class="header__logo" src="assets/header/Ellipse_1_copy_11.png" alt="Ellips">
+    <div class="header__text-block">
+      <h1 class="header__text-block_main">Working</h1>
+      <h1 class="header__text-block_main-bottom">with design lab</h1>
```
Comment (vadeneev) [inline doc1/public/index.html]: We should not use more then one h1 tag - it is recommended to follow this rule, as far as h1 is one and most important on the page

File: doc1/public/index.html [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+        design and product management.
+      </h2>
+    </div>
+    <a href="#aboutUs" class="header__wrapper-mouse-icon">
+      <img class="header__mouse-icon" src="assets/header/Shape_1.svg" alt="mouse icon">
+    </a>
+  </header>
+
+  <section class="section" id="aboutUs">
+    <div class="section__text-block section__text-block_bordered-black">
```
Comment (vadeneev) [inline doc1/public/index.html]: black / white - ok, what if tomorrow UX team will tell you to start use dark/light gray instead ?

File: doc1/public/index.html [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+      <h3 class="section__header">About us</h3>
+      <article class="section__article section__article_abous-us-top">
```
Comment (vadeneev) [inline doc1/public/index.html]: article supposed and recommended to have header https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
In that particular case articles applied incorrectly

File: doc1/public/index.html [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+        the user’s needs and the product requirements.
+        Then we help translate them into effective and elegant solutions.
+      </article>
+    </div>
+  </section>
+
+  <section class="section section_services">
+    <div class="section__text-block section__text-block_bordered-white">
+      <h3 class="section__header">Services</h3>
+      <article class="section__article section__article_services-top">
```
Comment (vadeneev) [inline doc1/public/index.html]: So what is modifier means here ? services-top mean it should be on top position, mean position_top ?

File: doc1/public/index.html [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+      </article>
+      <article class="section__article section__article_services-bottom">
+        The outcome of this scoping session is a proposal or high-level statement of work.
+        Based on your requirements, we can provide a complete offering or just the pieces you need.
+      </article>
+    </div>
+  </section>
+
+  <section class="section">
+    <div class="section__text-block section__text-block_bordered-black section__text-block_what-we-do">
```
Comment (vadeneev) [inline doc1/public/index.html]: Why is that block so special, that it has modifier _what-we-do ?

**Review by vadeneev — CHANGES_REQUESTED** [review]

File: doc1/public/index.html [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+  </header>
+
+  <section class="section" id="aboutUs">
+    <div class="section__text-block section__text-block_bordered-black">
+      <h3 class="section__header">About us</h3>
+      <p class="section__article section__article_abous-us section__article_abous-us-position-top">
+        When you work with us, you are part of a team of committed collaborators.
+        All aspects of our software development process are integrated, iterative and agile.
+      </p>
+      <p class="section__article section__article_abous-us section__article_abous-us-position-bottom">
```
Comment (vadeneev) [inline doc1/public/index.html]: abous-us ?

File: doc1/public/index.html [hunk 18]
```diff
@@ -0,0 +1,164 @@
...
+      <p class="section__article section__article_services section__article_services-position-bottom">
+        The outcome of this scoping session is a proposal or high-level statement of work.
+        Based on your requirements, we can provide a complete offering or just the pieces you need.
+      </p>
+    </div>
+  </section>
+
+  <section class="section">
+    <div class="section__text-block section__text-block_bordered-black">
+      <h3 class="section__header section__header_what-we-do">What we do</h3>
```
Comment (vadeneev) [inline doc1/public/index.html]: Are you sure that it is fine name for modifier ?

**Review by vadeneev — CHANGES_REQUESTED** [review]

File: doc1/public/index.html (hunk 13 above)
Comment (vadeneev) [reply doc1/public/index.html]: Not sure that this one was resolved

**Review by vadeneev — CHANGES_REQUESTED** [review]

File: doc1/public/index.html [hunk 19]
```diff
@@ -0,0 +1,163 @@
...
+        the user’s needs and the product requirements.
+        Then we help translate them into effective and elegant solutions.
+      </p>
+    </div>
+  </section>
+
+  <section class="section section_services">
+    <div class="section__text-block section__text-block_bordered-primary-color">
+      <h3 class="section__header">Services</h3>
+      <p class="section__article section__article_services section__article_position-top">
```
Comment (alexkond) [inline doc1/public/index.html]: Can't find usage of "section__article_services" class in CSS, looks like it's obsolete

File: doc1/public/index.html [hunk 20]
```diff
@@ -0,0 +1,163 @@
...
+        Our approach unifies design, development and
+        product management to create exceptional products.
+        We start each project by understanding your business goals,
+        the user’s needs and the product requirements.
+        Then we help translate them into effective and elegant solutions.
+      </p>
+    </div>
+  </section>
+
+  <section class="section section_services">
```
Comment (alexkond) [inline doc1/public/index.html:46]: Can't find usage of "section_services" class in CSS, looks like it's obsolete

File: doc1/public/style/style.css [hunk 21]
```diff
@@ -0,0 +1,435 @@
...
+}
+
+.section__text-block {
+  box-sizing: border-box;
+  width: 43%;
+  display: flex;
+  flex-direction: column;
+}
+
+.section__text-block {
```
Comment (alexkond) [inline doc1/public/style/style.css]: You have a rule for ".section__text-block" class right above and it partially does the same things like this one.

File: doc1/public/style/style.css [hunk 22]
```diff
@@ -0,0 +1,435 @@
...
+  right: 0;
+  width: 200px;
+  position: absolute;
+  content: "";
+  border-top-width: 2px;
+  border-top-style: solid;
+  border-top-color: inherit;
+}
+
+.section__text-block_bordered-secondary-color {
```
Comment (alexkond) [inline doc1/public/style/style.css]: This class changes not only borders but color also, consider renaming it.

**Review by alexkond — CHANGES_REQUESTED** [review]

File: doc1/public/index.html (hunk 19 above)
Comment (EasileleM) [reply doc1/public/index.html]: It sets up background image to this section.

File: doc1/public/index.html (hunk 20 above)
Comment (EasileleM) [reply doc1/public/index.html]: section_services

Comment (EasileleM) [reply doc1/public/index.html:46]: It sets up background image to this section

**Review by vovcyan — DISMISSED** [review]

**Review by alexkond — DISMISSED** [review]

**Review by vadeneev — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**